### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesNew_NewRecipe_ScreenReader: Not able to adjust the slider value when Voiceover is turned on.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15320]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_iOS_May23; #.NET Core; #WCAG1.3.1; #iOS; #iPhone; #Voiceover;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: iOS (iPhone 6s)
Screen Reader: Voiceover

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Start Voice over.
6. Swipe till slider and observe that not able to adjust the slider value when Voiceover is turned on.

### Actual Results:
Not able to adjust the slider value when Voiceover is turned on.

**Similar issue is also observed in below scenario:
Issue 1: Recipes App My Recipes Edit_ Edit Recipe:**

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till any existing recipe and double tap.
4. "Recipe" Screen will open.
5. Swipe till "Edit" button and double tap.
6. Start Voice over.
7. Swipe till slider and observe that not able to adjust the slider value when Voiceover is turned on.

### Expected Results:
User should be able to adjust the slider value when Voiceover is turned on.

### User Impact:
Screen reader users will face difficulties while adjusting the slider value and will be blocked here.

### Attachment:
![Presentation8](https://github.com/dotnet/maui/assets/90901519/10a35817-d118-446f-b323-b4bf00759d07)

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS version 15.7.6

**Comment by PoojaNamde** [comment]
@rachelkang 

Could you please provide the video again as we are not able to see the video and way of navigation that you used to enable slider adjustment.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-21-23;

Verified this issue in the application, issue is not able to repro as when we navigate using upward and downward swipe. Please find below attachment for reference.


https://github.com/dotnet/maui/assets/90901519/2c0a489c-7a03-4615-8a9b-00799fbebac9
